ui/app/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -19,7 +19,6 @@ export default class ConfirmPageContainerContent extends Component {
     assetImage: PropTypes.string,
```

Comment:
Going to add review and comments file by file, to ensure my own thinking is good.